src/routers/routerV2.ts

```diff
@@ -26,6 +37,13 @@ routerV2.get(
 
 // Speech-to-Text
 routerV2.post('/speech-to-text', analytics, validateApiKey, getTranscription);
+routerV2.post(
+  '/igbo-to-english',
+  translationRateLimiter,
+  analytics,
+  validateApiKey,
```

Comment:
we can talk about this more on our call, but you will want to update `authorizeDeveloperUsage` so that it will also count usage for the currently used API Key. 

You can look at `ApiUsageLimit.ts` which includes all the usage limits we allow for regular developer accounts.